### DISCUSSION TITLE
<title>is more easy get the Quidraw

### DISCUSSION THREAD

**Discussion by michael34df** [community/community#64353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

is more easy get the Quidraw?

**Comment by santimars** [comment]
yes just read this documentation bro
https://github.com/Thinkright20/Profile-Badges

**Reply by michael34df** [reply]
thanks

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
